src/lib/media-import/status.ts

```diff
@@ -271,44 +273,120 @@ ${ maybeExitPrompt }
 			void checkStatus( IMPORT_MEDIA_PROGRESS_POLL_INTERVAL );
 		} );
 
+	async function exportFailureDetails(
+		fileErrors: Maybe< AppEnvironmentMediaImportStatusFailureDetailsFileErrors >[]
+	) {
+		const formattedData = buildFileErrors( fileErrors, exportFileErrorsToJson );
+		const errorsFile = `media-import-${ app.name ?? '' }-${ Date.now() }${
+			exportFileErrorsToJson ? '.json' : '.txt'
+		}`;
+		try {
+			await writeFile( errorsFile, formattedData );
+			progressTracker.suffix += `${ chalk.yellow(
+				`⚠️  All errors have been exported to ${ chalk.bold( resolve( errorsFile ) ) }`
+			) }`;
+		} catch ( writeFileErr ) {
+			progressTracker.suffix += `${ chalk.red(
+				`Could not export errors to file\n${ ( writeFileErr as Error ).message }`
+			) }`;
+		}
+	}
+
+	async function fetchFailureDetails( failureDetailsUrl: string ) {
+		progressTracker.suffix += `
+=============================================================
+Downloading errors details from ${ failureDetailsUrl }...
+\n`;
+		try {
+			const response = await fetch( failureDetailsUrl );
+			return ( await response.json() ) as AppEnvironmentMediaImportStatusFailureDetailsFileErrors[];
+		} catch ( err ) {
+			progressTracker.suffix += `${ chalk.red(
+				`Could not download file import errors report\n${ ( err as Error ).message }`
+			) }`;
+			throw err;
+		}
+	}
+
+	async function promptFailureDetailsDownload( failureDetailsUrl: string ) {
+		progressTracker.suffix += `${ chalk.yellow(
+			`⚠️  Error details can be found on ${ chalk.bold(
+				failureDetailsUrl
+			) } ${ chalk.italic.yellow(
+				'(This link will be valid for the next 15 minutes. The report is retained for 7 days since the import was run.)'
```

Comment:
```suggestion
				'(This link will be valid for the next 15 minutes. The report is retained for 7 days from the completion of the import.)'
```

maybe this is a tad better? idk

Reply:
Maybe "This link will be valid for the next 15 minutes. The report is retained for 7 days from the time the import was run." ?